aemedge/blocks/channel-lookup/channel-lookup.js

```diff
@@ -0,0 +1,114 @@
+import { createTag } from '../../scripts/utils.js';
+
+const ZIPCODE_ENDPOINT = 'https://p-geo.movetv.com/geo';
+const DEFAULT_ZIPCODE = '90020';
+const ZIPCODE_KEY = 'user_zip';
+let gqlResponse = '';
+
+export async function getZipcode() {
+  let zipcode = localStorage.getItem(ZIPCODE_KEY);
+  if (!zipcode) {
+    const response = await fetch(ZIPCODE_ENDPOINT);
+    const data = await response.json();
+    zipcode = data?.zip_code || DEFAULT_ZIPCODE;
+    localStorage.setItem(ZIPCODE_KEY, zipcode);
+  }
+  return zipcode;
+}
+function showWarningMessage() {
+  return `
+    <div type="warning" class="warning channel-data">
+      <img src="/aemedge/icons/warning-icon.svg" id="icon" class="warning-icon"/>
+      <h3 class="warning-heading">
+      <span>
+      <p class="warning-msg">Local programming
+      is not available with Sling in your area. You can still watch the Olympics on USA on-demand.
+      </p>
+      </span>
+      </h3>
+    </div>`;
+}
+
+function channelName(packages) {
+  let channelDiv;
+  packages.forEach((pkg) => {
+    if (channelDiv !== undefined) {
+      channelDiv += `<picture><img class="channel-img"
```

Comment:
I suggest to load images from google drive so that the docs and icons can be managed and loaded from Edge Delivery Services . The icons are available at - https://drive.google.com/drive/folders/1gT4GX2V18zwOG3gi0Z1i-orVfJlrH6bw and if they are missing you can a)upload them here b)preview , publish and use them. You can check Channel Shopper , Base Cards and Package Cards block's HTML for reference . They are used on Home Page and Sports page

Reply:
done, please review